### PR TITLE
Translated Idempotence term to pt-br

### PR DESCRIPTION
**Pull request by matheuspaes**

### CODE REVIEW

File: content/pt-br/idempotence.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Idempotência
+status: Completed
+category: propriedade
+tags: ["infrastructure", "architecture", ""]
```
Comment (edsoncelio) [inline content/pt-br/idempotence.md:5]: You can translate the tags :)

**Review by edsoncelio — COMMENTED** [review]
Hi @matheuspaes, thank for your PR!
I left a small comment and another thing: could you sign your commits just to pass in the DCO check?

**Review by edsoncelio — APPROVED** [review]
LGTM

**Review by brunoguidone — APPROVED** [review]
LGTM

**Pull request merged**